pydm/utilities/remove_protocol.py

```diff
@@ -37,8 +37,19 @@ def protocol_and_address(address):
     match = re.match('.*?://', address)
     protocol = None
     addr = address
+    subfield = None 
+
     if match:
         protocol = match.group(0)[:-3]
         addr = address.replace(match.group(0), '')
+        resulting_string = addr.split('/', 1)
+        
+        if len(resulting_string) < 2:
+            resulting_string.append(None)
+        
+        addr, subfield = resulting_string
+
+        if subfield:
+            subfield = subfield.split('/')
 
-    return protocol, addr
+    return protocol, addr, subfield
```

Comment:
(Just an observation as I saw this PR go by; sorry if in-draft suggestions are annoying)

Might be worth considering `urllib.parse.urlparse` here?
```python
In [1]: import urllib

In [2]: urllib.parse.urlparse("https://a.b/c/d?e=f#ghi")
Out[2]: ParseResult(scheme='https', netloc='a.b', path='/c/d', params='', query='e=f', fragment='ghi')
```

Reply:
Thanks, yeah that is a good idea, it's what is used in the calc and local plugins if I recall. 

I was going with a design idea Matt suggested in https://github.com/slaclab/pydm/pull/979 but I think '/' seems to be valid in a PV name, so I will have to think a bit more on how pass the NTTable PV's sub-field